network-config-analyzer/CalicoNetworkPolicy.py

```diff
@@ -75,7 +75,8 @@ def __init__(self, name, namespace):
         self.order = None  # None means infinity here
 
     def __eq__(self, other):
-        return super().__eq__(other) and self.order == other.order
+        return super().__eq__(other) and \
+            (not hasattr(other, 'order') or self.order == other.order)
```

Comment:
For the case of mixed Calico/Ingress policies (Ingress policy does not have 'order' attribute)

Reply:
Maybe `isinstance(other, CalicoNetworkPolicy) and super().__eq__(other)` would be more intuitive?
I wouldn't expect `other` to be equal in case it does not have an `order` attribute.

Reply:
I agree, it's more readable. Changed.